OracleHTTPServer/dockerfiles/12.2.1.4.0/container-scripts/provisionOHS.sh

```diff
@@ -41,12 +40,12 @@ echo "OHS_COMPONENT_NAME=${OHS_COMPONENT_NAME:?"Please set OHS_COMPONENT_NAME"}"
 export MW_HOME ORACLE_HOME DOMAIN_NAME OHS_COMPONENT_NAME
 
 
-#Set WL_HOME, WLST_HOME, DOMAIN_HOME and NODEMGR_HOME
-WL_HOME=${ORACLE_HOME}/wlserver
+#Set WL_HOME, WLST_HOME, DOMAIN_HOME, NODEMGR_HOME, and LOGS_DIR
+#WL_HOME=${ORACLE_HOME}/wlserver
 WLST_HOME=${ORACLE_HOME}/oracle_common/common/bin
 echo "WLST_HOME=${WLST_HOME}"
 
-DOMAIN_HOME=${ORACLE_HOME}/user_projects/domains/${DOMAIN_NAME}
+DOMAIN_HOME=${ORACLE_HOME}/ohssa/user_projects/domains/${DOMAIN_NAME}
```

Comment:
The value of $ORACLE_HOME is /u01/oracle/ohssa. So, I think that the old value was correct.

Reply:
made ORACLE_HOME=/u01/oracle